### DISCUSSION TITLE
Unable to serve Jupyter Lite from Windows 10 terminal

### DISCUSSION THREAD

**Discussion by jazl** [jupyterlite/jupyterlite#591] [Q&A] [answered]
After installing Miniconda for Windows, I followed the steps in the Authoring Server Extensions tutorial. But when serving up the website from a Windows 10 Anaconda prompt, I receive the following error when the HTML page tries to import the config-utils.js file:

```
Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/plain". Strict MIME type checking is enforced for module scripts per HTML spec.

Uncaught TypeError: Failed to fetch dynamically imported module: http://localhost:8000/config-utils.js
```
Apparently the reason is because the Content-Type of the response is "text/plain" instead of "application/javascript", as shown in the pic below.

This only occurs when serving from the Anaconda Prompt (miniconda3) or the Anaconda Powershell Prompt. It works fine when serving from a Windows Subsystem for Linux Ubuntu terminal from the same machine. Has anyone else encountered this?

![image](https://user-images.githubusercontent.com/2700780/162084233-b5e807ca-ae00-4553-847f-f0cb6f1b1a9e.png)

**Comment by bollwyvl** [comment, marked as answer]
This is a python `wontfix` bug: python defers to your registry, which has had a mapping for `.js` added by some other program, see also: https://github.com/jupyterlite/jupyterlite/issues/569

This is _possible_ to be broken on a POSIX system like WSL, but much less likely, as programs seldom override things like that.

We may well have to do some broader-reaching custom MIME stuff, so this could be addressed as a default, but this is seldom the _only_ thing that will break.
